src/data/auth/authServices.ts

```diff
@@ -1,3 +1,52 @@
+import {
+  SignUpUserData,
+  SignUnUserResponse,
+  SignInUserData,
+  SignInUserResponse,
+  RefreshTokenData,
+  RefreshTokenResponse,
+  LogoutUserResponse,
+} from '../../types/auth/auth';
+import axiosInstance from '../http/http';
 import { signInUserMock } from './__mocks/authServicesMocks';
 
 export const signInUser = signInUserMock;
+
+export const signUpUser = (
+  signUpUserData: SignUpUserData,
+): Promise<SignUnUserResponse> => {
+  return axiosInstance
+    .post<SignUnUserResponse>(`signup`, signUpUserData)
+    .then((res) => {
+      return res.data;
+    });
+};
```

Comment:
I prefer to use async/await syntax. It help to organize the code and make code more clarify and structural. Can you please update the all services using this syntax?
Example:
```suggestion
export const signUpUser = async (
  signUpUserData: SignUpUserData,
): Promise<SignUnUserResponse> => {
  const { data } =  await axiosInstance.post<SignUnUserResponse>(`signup`, signUpUserData);
   return data;
};
```